### ISSUE TITLE
UPnP-IGD

### ISSUE THREAD

**Issue by kgersen** [LaFibre-info/Freepro#16]
Pas possible d'activer UPnP-IGD sur la box pour permettre l'ouverture des ports par programmation/application.

**Comment by phastier** [comment]
Sur une box Pro, et vu les risques de sécurité que ça peut représenter, ça ne me choque pas forcément.
Mais laisser le choix serait une bonne chose en tout cas.

**Comment by rmanus** [comment]
Effectivement, pour des raisons de sécurité cette feature n'est pas à l'ordre du jour.
